### PR TITLE
Refactor Backing Services

### PR DESCRIPTION
**Pull request by dcasavant**
* ~~Move Constellation from the `/pega` charts folder to `/backingservices`. **This change may require a major version roll of the Charts**~~
* Deprecate the single-tenant Constellation deployment under the `/pega` charts folder and add a new copy under `/backingservices` for future use. The deprecated chart can be removed with the next major version update.
* Updated the backing services readme to refactor SRS content into the SRS subchart readme.
* Establish UIServices as a team with CODEOWNERS control over the constellation subchart.

### CODE REVIEW

File: charts/backingservices/README.md
Comment (taz-pega-work) [inline charts/backingservices/README.md:3]: minor suggestion to get rid of passive voice>>
You deploy dedicated services to support Pega Infinity as an independent microservice using the Pega Infinity backing service Helm charts. To use these services, deploy the backing services chart independently following the instructions below.

**Review by taz-pega-work — APPROVED** [review]
Looks good.

**Review by taz-pega-work — APPROVED** [review]

**Review by npjohnson637 — APPROVED** [review]

File: charts/backingservices/README.md
Comment (dcasavant) [reply charts/backingservices/README.md]: Updated

**Comment by anuds20** [comment]
@dcasavant we have a PR with changes related to these charts 

https://github.com/pegasystems/pega-helm-charts/pull/415

Since both PRs have changes related to the constellation-static service charts can we club both of these PRs and create a new one? If so I can make the same changes in your branch and we can merge all the changes at once.

**Review by taz-pega-work — APPROVED** [review]
I'll approve if it helps you merge...I can't verify that the code is accurate

**Comment by anuds20** [comment]
@dcasavant I will be merging couple of changes to this PR soon. In case you plan to merge this please wait for the changes. I will update the comment thread once I have pushed the changes.

File: docs/prepping-local-system-runbook-linux.md
Comment (taz-pega-work) [inline docs/prepping-local-system-runbook-linux.md:441]: @khick77 should confirm that the docker image details should be removed from the linux and windows prepping runbooks. We moved all of these details in August 2022 to>https://docs.pega.com/client-managed-cloud/87/pega-provided-docker-images

I can update the existing wording there to use this new link>>
The docker image format is different for backing services and does not follow the datestamp pattern. For installations of or upgrades to Pega Infinity 8.6, you may configure Docker image `platform-services/search-n-reporting-service` with `<srs-version>` compatible with the Pega version as provided in [compatibility matrix](../charts/backingservices/charts/srs/README.md#srs-version-compatibility-matrix).

File: docs/prepping-local-system-runbook-windows.md
Comment (taz-pega-work) [inline docs/prepping-local-system-runbook-windows.md:267]: @khick77 should confirm that the docker image details should be removed from the linux and windows prepping runbooks. We moved all of these details in August 2022 to>https://docs.pega.com/client-managed-cloud/87/pega-provided-docker-images

I can update the existing wording there to use this new link>>
The docker image format is different for backing services and does not follow the datestamp pattern. For installations of or upgrades to Pega Infinity 8.6, you may configure Docker image `platform-services/search-n-reporting-service` with `<srs-version>` compatible with the Pega version as provided in [compatibility matrix](../charts/backingservices/charts/srs/README.md#srs-version-compatibility-matrix).

**Review by taz-pega-work — COMMENTED** [review]
please note that the docker image details should no longer be in these prepping runbooks. @khick77 please confirm that you had a PR to remove this information since we are maintaining this info on docs.pega.com

File: docs/prepping-local-system-runbook-windows.md
Comment (khick77) [reply docs/prepping-local-system-runbook-windows.md:267]: @taz-pega-work 
My PR is here:
https://github.com/pegasystems/pega-helm-charts/pull/389/files
it was merged, and only affected the readme.md file.  Dave's changes to that file seem to be putting back in the notes that I had ripped out.  I'll leave it to you to decide if that's what you want or not.  It sounds like not.

Comment (taz-pega-work) [reply docs/prepping-local-system-runbook-windows.md:267]: we don't want the information to live in two places. Clients need to use this information and I think the goal of your PR was to also take out the detailed Docker image information out of each prepping runbook and the readme and the two runbooks would then just point to the latest information. For 8.7 that's >>https://docs.pega.com/client-managed-cloud/87/pega-provided-docker-images
and for 8.8, it will be here for 8.8 GA>https://docs.pega.com/bundle/platform-88/page/platform/deployment/client-managed-cloud/pega-docker-images-manage.html
since that won't be published until mid-October, you can see the content here until then>>https://pega-prod.zoominsoftware.io/bundle/platform-88/page/platform/deployment/client-managed-cloud/pega-docker-images-manage.html
I think for the 8.8 timeframe, we need to have Dave's changes go into 8.8 version, since right now, that directory structure refers to won't be live until around 8.8 GA. 

Does this make sense??

**Review by anuds20 — APPROVED** [review]

File: README.md
Comment (taz-pega-work) [inline README.md:171]: please update this link to use the one we will have for the 8.8 release>>
see [Pega-provided Docker images](https://docs.pega.com/bundle/platform-88/page/platform/deployment/client-managed-cloud/pega-docker-images-manage.html)

Note: you'll see the current doc at https://pega-prod.zoominsoftware.io/bundle/platform-88/page/platform/deployment/client-managed-cloud/pega-docker-images-manage.html which will automatically update to https://docs.pega.com/bundle/platform-88/page/platform/deployment/client-managed-cloud/pega-docker-images-manage.html when the 8.8 docs go live at GA

File: README.md
Comment (taz-pega-work) [inline README.md:153]: this list of images is missing the Hazelcast image>>platform/clustering-service

This was the detail added by Varun's team a while ago...

`platform/clustering-service`  | (Download required) Deploys the required clustering service(Hazelcast) for the caching capabilities in Pega platform as a backing service. This Docker image contains Hazelcast clustering Service. <br/><br/>*Note: This feature is supported only from platform version "8.6" to deploy the platform in client-server Hazelcast model. Embedded deployment would not be supported in future platform releases.*| `<version>` |

**Review by pega-talba — APPROVED** [review]

**Review by pega-talba — APPROVED** [review]

**Review by APegaDavis — APPROVED** [review]

**Review by misterdorito — APPROVED** [review]

File: README.md
Comment (taz-pega-work) [inline README.md:160]: I'd recommend modifying these details to be closer to 8.8 and October 2022 so it doesn't look so stale>>

For the `platform/installer` image, the :tag represents the version of Pega you want to install, for example the tag :8.8.0 will install Pega Platform version 8.8.0 (the GA version).
For `platform/pega` and `platform/search` images, Pega also offers an image with a version tag appended with a datestamp using the pattern `pegaVersion-YYYYMMDD` to indicate the version and the date that Pega built the image. For example, if you pull the `platform/pega` with a tag, `pega:8.5.1-20221018`, the tag indicates that Pega built this 8.8.0 image on 18 October 2022. Using the version tag without the datestamp will always point to the most recently built image for that version.

**Review by taz-pega-work — CHANGES_REQUESTED** [review]
I'll review if there's any new information here around the images and put them into my client-facing docs on docs.pega.com by the end of this week so we are sync'd.

**Review by taz-pega-work — APPROVED** [review]

**Pull request merged**
